src/Behat/Config/Filter/NameFilter.php

```diff
@@ -4,20 +4,13 @@
 
 namespace Behat\Config\Filter;
 
-final class NameFilter implements FilterInterface
+final class NameFilter extends Filter
```

Comment:
The named filters now just provide a name to the parent class and leave the rest of the implementation to this class